packages/nx/src/command-line/nx-commands.ts

```diff
@@ -810,37 +810,35 @@ async function withCustomGeneratorOptions(
   const options = [];
   const positionals = [];
 
-  Object.entries(schema.properties as WorkspaceGeneratorProperties).forEach(
-    ([name, prop]) => {
-      options.push({
-        name,
-        definition: {
-          describe: prop.description,
-          type: prop.type,
-          default: prop.default,
-          choices: prop.enum,
-        },
-      });
-      if (isPositionalProperty(prop)) {
-        positionals.push({
+  if (schema.properties) {
+    Object.entries(schema.properties as WorkspaceGeneratorProperties).forEach(
```

Comment:
Rather than wrapping whole block in if statement, can we do `schema.properties ?? {}`?